skrf/network.py

```diff
@@ -3230,6 +3230,10 @@ def renumber(self, from_ports: Sequence[int], to_ports: Sequence[int]) -> None:
             List of port indices to change. Size between 1 and N_ports.
         to_ports : list-like
             List of desired port indices. Size between 1 and N_ports.
+        only_z0 : bool
+            If true only z0 is renumbered, s-parameters are not affected.
+            This should only be used after executing the "connect_s" method
+            which keeps the port index where you expect it to be.
```

Comment:
Could add an example using the only_z0 in the example section of the docstring?

Reply:
I have added a new case only_z0=True to the docstring.
Additionally, I attempted to resolve the LC Filter image error on the [Circuit tutorial page](https://scikit-rf.readthedocs.io/en/latest/tutorials/Circuit.html).  I apologize for inadvertently introducing this issue, I suspect the issue arose because both the Circuit reduction images and the original LC Filter images share the same name, `![low pass filter]`. This discrepancy does not occur in my local notebook environment.